test/confirmations_create_confirmation_request_unittest.cc

```diff
@@ -0,0 +1,170 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+
+#include "confirmations_client_mock.h"
+#include "brave/vendor/bat-native-confirmations/src/confirmations_impl.h"
```

Comment:
We shouldn't be using header paths like this, it indicates a problem with the `include_dirs` in the config. We should expose the internal-config to tests so they can access these files using the correct paths

Reply:
actually I already did that so not sure how these paths came back

Reply:
these paths came back as part of unit tests and I used an older template for the unit tests, let me fix these issues

Reply:
fixed 👍